src/libraries/System.Drawing.Common/src/System/Drawing/DrawingComWrappers.cs

```diff
@@ -0,0 +1,331 @@
+            wrapperEntry->Vtable = iStreamVtbl;
+            return wrapperEntry;
+        }
+
+        protected override unsafe ComInterfaceEntry* ComputeVtables(object obj, CreateComInterfaceFlags flags, out int count)
+        {
+            Debug.Assert(obj is Interop.Ole32.IStream);
+            Debug.Assert(s_wrapperEntry != null);
+
+            // Always return the same table mappings.
+            count = 1;
+            return s_wrapperEntry;
+        }
+
+        protected override object CreateObject(IntPtr externalComObject, CreateObjectFlags flags)
+        {
+            Debug.Assert(flags == CreateObjectFlags.None);
+
+            Guid pictureIID = IPicture.IID;
+            int hr = Marshal.QueryInterface(externalComObject, ref pictureIID, out IntPtr comObject);
+            if (hr == 0)
+            {
+                return new PictureWrapper(comObject);
+            }
+
+            throw new NotImplementedException();
+        }
+
+        protected override void ReleaseObjects(IEnumerable objects)
+        {
+            throw new NotImplementedException();
+        }
+
+        internal static class IStreamVtbl
+        {
+            public static IntPtr Create(IntPtr fpQueryInteface, IntPtr fpAddRef, IntPtr fpRelease)
+            {
+                IntPtr* vtblRaw = (IntPtr*)RuntimeHelpers.AllocateTypeAssociatedMemory(typeof(IStreamVtbl), IntPtr.Size * 14);
+                vtblRaw[0] = fpQueryInteface;
+                vtblRaw[1] = fpAddRef;
+                vtblRaw[2] = fpRelease;
+                vtblRaw[3] = (IntPtr)(delegate* unmanaged<IntPtr, byte*, uint, uint*, int>)&ReadImplementation;
+                vtblRaw[4] = (IntPtr)(delegate* unmanaged<IntPtr, byte*, uint, uint*, int>)&WriteImplementation;
+                vtblRaw[5] = (IntPtr)(delegate* unmanaged<IntPtr, long, SeekOrigin, ulong*, int>)&SeekImplementation;
+                vtblRaw[6] = (IntPtr)(delegate* unmanaged<IntPtr, ulong, int>)&SetSizeImplementation;
+                vtblRaw[7] = (IntPtr)(delegate* unmanaged<IntPtr, IntPtr, ulong, ulong*, ulong*, int>)&CopyToImplementation;
+                vtblRaw[8] = (IntPtr)(delegate* unmanaged<IntPtr, uint, int>)&CommitImplementation;
+                vtblRaw[9] = (IntPtr)(delegate* unmanaged<IntPtr, int>)&RevertImplementation;
+                vtblRaw[10] = (IntPtr)(delegate* unmanaged<IntPtr, ulong, ulong, uint, int>)&LockRegionImplementation;
+                vtblRaw[11] = (IntPtr)(delegate* unmanaged<IntPtr, ulong, ulong, uint, int>)&UnlockRegionImplementation;
+                vtblRaw[12] = (IntPtr)(delegate* unmanaged<IntPtr, out Interop.Ole32.STATSTG, Interop.Ole32.STATFLAG, int>)&StatImplementation;
+                vtblRaw[13] = (IntPtr)(delegate* unmanaged<IntPtr, IntPtr*, int>)&CloneImplementation;
+
+                return (IntPtr)vtblRaw;
+            }
+
+            [UnmanagedCallersOnly]
+            private static int ReadImplementation(IntPtr thisPtr, byte* pv, uint cb, uint* pcbRead)
```

Comment:
Is the `Implementation` suffix on all methods still necessary?

Reply:
Not anymore, now that the fields are removed. Before the names collided.